tests/components/miele/snapshots/test_sensor.ambr

```diff
@@ -710,6 +710,7 @@
         'spin',
         'starch_stop',
         'steam_smoothing',
+        'unknown',
```

Comment:
unknown shouldn't be a valid state in the options list though

Reply:
Yes and no. I want to differ between unknown = "None for whatever reason" and unknown_code. My idea was to translate to Unknown code in strings.json. If we take this unknown out of the options an error will be generated whenever the state is "unknown". Perhaps the re-mapping should be changed to "unknown_code"?

Reply:
But what benefit does the user have in knowing that we don't know the state? In the end we still don't know what the state is

Reply:
There is a distinction between unknown state from the API and known state/code but missing translation. It will aid troubleshooting to use specific terms.

Reply:
"unknown" shouldn't be a state used by any integration. The distinction you are trying to make, doesn't exist in the resulting state.

Reply:
OK. Changed accordingly.